### PR TITLE
KAFKA-2344; kafka-merge-pr improvements

### PR DESCRIPTION
**Pull request by ijuma**
The first 4 commits are adapted from changes that have been done to the Spark version and the last one is the feature that @gwenshap asked for.

### CODE REVIEW

File: kafka-merge-pr.py [hunk 1]
```diff
@@ -130,7 +130,15 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc):
...
                               key=lambda x: commit_authors.count(x), reverse=True)
-    primary_author = distinct_authors[0]
+    primary_author = raw_input(
+        "Enter primary author in the format of \"name <email>\" [%s]: " %
+        distinct_authors[0])
+    if primary_author == "":
+        primary_author = distinct_authors[0]
+
+    reviewers = raw_input(
+        "Enter reviewers in the format of \"name1 <email1>\", \"name2 <email2>\: ").strip()
```
Comment (gwenshap) [inline kafka-merge-pr.py]: super nit: the string is slightly malformatted.  The script displays:
Enter reviewers in the format of "name1 <email1>", "name2 <email2>:

Also, Its not clear if I should actually type the quotes when entering reviewers.

Comment (ijuma) [inline kafka-merge-pr.py]: Good catch. I was trying to follow the format for `primary author`, but made a mistake. Fixed it to quote the whole bit of text. Do you think it's clearer now?

**Comment by b-goyal** [comment]
The first 4 commits are adapted from changes that have been done to the Spark version and the last one is the feature that @gwenshap asked for.

**Comment by b-goyal** [comment]
The first 4 commits are adapted from changes that have been done to the Spark version and the last one is the feature that @gwenshap asked for.

**Pull request closed without merging**